src/main/java/com/google/api/codegen/discovery/config/ApiaryConfigToSampleConfigConverter.java

```diff
@@ -104,7 +106,7 @@ private MethodInfo createMethod(Method method) {
     boolean isPageStreaming = isPageStreaming(method);
     FieldInfo pageStreamingResourceField = null;
     if (isPageStreaming) {
-      Type containerType = apiaryConfig.getType(responseTypeUrl);
+      Type containerType = apiaryConfig.getType(method.getResponseTypeUrl());
```

Comment:
It... doesn't actually do anything really, I think this was left over from my work and I forgot to change it. Reverted